docs/core/extensions/configuration-providers.md

```diff
@@ -25,6 +25,9 @@ Configuration in .NET is possible with configuration providers. There are severa
 - [XML configuration provider](#xml-configuration-provider)
 - [INI configuration provider](#ini-configuration-provider)
 
+> [!IMPORTANT]
+> Keys are case-insensitive, so be sure that your keys are unique. All of the file configuration providers throw the <xref:System.FormatException> when duplicate keys are found.
```

Comment:
Why does this call for `[!IMPORTANT]`. IMAO, the blazing chrome of something so obvious makes me want to skip other such chrome.

And I don't see the connection between case-insensitive, ***so***
And unique across what?  It's expected you use the same key in multiple config providers. The connection string value in debug is different than the value in production.

`Keys are case-insensitive` that's the important part this doc was missing.